base/dcp_feed.go

```diff
@@ -143,8 +144,26 @@ func (r *DCPReceiver) OnError(err error) {
 	// r.notify(bucketName, err)
 }
 
+// Only a subset of Sync Gateway's internal documents need to be included during DCP processing: user, role, and
+// unused sequence documents.  Any other documents with the leading '_sync' prefix can be ignored.
+// Returns true for documents that should be processed, false for those that do not need processing.
+//  TODO: The hardcoded strings here should be changed to constants (CBG-274)
+func dcpKeyFilter(key []byte) bool {
+	if bytes.HasPrefix(key, []byte("_sync")) {
```

Comment:
Looks good. What do you think about removing the nesting of ifs, by reversing the first condition?

E.g:
```go
if !bytes.HasPrefix(key, []byte("_sync")) {
	return true
}

if bytes.HasPrefix(key, []byte("_sync:unused")) ||
	bytes.HasPrefix(key, []byte("_sync:user:")) ||
	bytes.HasPrefix(key, []byte("_sync:role:")) {
	return true
}

// other _sync docs we don't care about
return false
```

Reply:
Agree - will make that change